docs/source/testing/testing.rst

```diff
@@ -0,0 +1,19 @@
+Running Tests
```

Comment:
Thanks for adding the doc here as well! Same comments apply as to the `CONTRIBUTING.md` file.